src/plugins/intel_gpu/include/intel_gpu/primitives/experimental_detectron_detection_output.hpp

```diff
@@ -0,0 +1,82 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+///////////////////////////////////////////////////////////////////////////////////////////////////
+#pragma once
+#include <utility>
+#include <vector>
+
+#include "primitive.hpp"
+
+namespace cldnn {
+/// @addtogroup cpp_api C++ API
+/// @{
+/// @addtogroup cpp_topology Network Topology
+/// @{
+/// @addtogroup cpp_primitives Primitives
+/// @{
+
+/// @brief experimental detectron detection output
+struct experimental_detectron_detection_output : public primitive_base<experimental_detectron_detection_output> {
+    CLDNN_DECLARE_PRIMITIVE(experimental_detectron_detection_output)
+
+    experimental_detectron_detection_output(const primitive_id& id,
+                                            const primitive_id& input_rois,
+                                            const primitive_id& input_deltas,
+                                            const primitive_id& input_scores,
+                                            const primitive_id& input_im_info,
+                                            const primitive_id& output_classes,
+                                            const primitive_id& output_scores,
+                                            float score_threshold,
+                                            float nms_threshold,
+                                            int64_t num_classes,
+                                            int64_t post_nms_count,
+                                            size_t max_detections_per_image,
```

Comment:
Any special reason to use big sized int like int64_t and size_t? I think int32_t is enough.

Reply:
Mainly I was just using the same types as in the [operation](https://github.com/openvinotoolkit/openvino/blob/master/src/core/include/openvino/op/experimental_detectron_detection_output.hpp#L35).

Changed to use int